[I 2025-06-29 17:08:43.738 ServerApp] jupyter_lsp | extension was successfully linked.
[I 2025-06-29 17:08:43.739 ServerApp] jupyter_server_terminals | extension was successfully linked.
[I 2025-06-29 17:08:43.741 ServerApp] jupyterlab | extension was successfully linked.
[I 2025-06-29 17:08:43.742 ServerApp] notebook | extension was successfully linked.
[I 2025-06-29 17:08:43.815 ServerApp] notebook_shim | extension was successfully linked.
[I 2025-06-29 17:08:43.862 ServerApp] notebook_shim | extension was successfully loaded.
[I 2025-06-29 17:08:43.863 ServerApp] jupyter_lsp | extension was successfully loaded.
[I 2025-06-29 17:08:43.863 ServerApp] jupyter_server_terminals | extension was successfully loaded.
[I 2025-06-29 17:08:43.864 LabApp] JupyterLab extension loaded from /Users/justinmartin/github/my_llm_project/.venv/lib/python3.13/site-packages/jupyterlab
[I 2025-06-29 17:08:43.864 LabApp] JupyterLab application directory is /Users/justinmartin/github/my_llm_project/.venv/share/jupyter/lab
[I 2025-06-29 17:08:43.864 LabApp] Extension Manager is 'pypi'.
[I 2025-06-29 17:08:43.884 ServerApp] jupyterlab | extension was successfully loaded.
[I 2025-06-29 17:08:43.885 ServerApp] notebook | extension was successfully loaded.
[I 2025-06-29 17:08:43.886 ServerApp] Serving notebooks from local directory: /Users/justinmartin/github/my_llm_project
[I 2025-06-29 17:08:43.886 ServerApp] Jupyter Server 2.16.0 is running at:
[I 2025-06-29 17:08:43.886 ServerApp] http://MacBook-Pro.local:8888/lab?token=4f7f0c06e8ebeac20583551280debc324bcc7f647ec40df7
[I 2025-06-29 17:08:43.886 ServerApp]     http://127.0.0.1:8888/lab?token=4f7f0c06e8ebeac20583551280debc324bcc7f647ec40df7
[I 2025-06-29 17:08:43.886 ServerApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 2025-06-29 17:08:43.887 ServerApp] 
    
    To access the server, open this file in a browser:
        file:///Users/justinmartin/Library/Jupyter/runtime/jpserver-21860-open.html
    Or copy and paste one of these URLs:
        http://MacBook-Pro.local:8888/lab?token=4f7f0c06e8ebeac20583551280debc324bcc7f647ec40df7
        http://127.0.0.1:8888/lab?token=4f7f0c06e8ebeac20583551280debc324bcc7f647ec40df7
[I 2025-06-29 17:08:43.968 ServerApp] Skipped non-installed server(s): bash-language-server, dockerfile-language-server-nodejs, javascript-typescript-langserver, jedi-language-server, julia-language-server, pyright, python-language-server, python-lsp-server, r-languageserver, sql-language-server, texlab, typescript-language-server, unified-language-server, vscode-css-languageserver-bin, vscode-html-languageserver-bin, vscode-json-languageserver-bin, yaml-language-server
[I 2025-06-29 17:09:07.103 ServerApp] 302 GET / (@127.0.0.1) 3.24ms
[I 2025-06-29 17:09:07.972 LabApp] Build is up to date
[W 2025-06-29 17:09:08.018 ServerApp] 404 GET /api/contents/my_llm_env3.12/lib/python3.12/site-packages/torch/utils/data/dataloader.py?content=0&hash=0&1751234948004 (990a2cfb35b74774811ecca73ce2f2a0@127.0.0.1) 0.81ms referer=http://localhost:8888/lab?
[W 2025-06-29 17:09:08.019 ServerApp] 404 GET /api/contents/my_llm_env3.12/lib/python3.12/site-packages/torch/utils/data/dataloader.py?content=0&hash=0&1751234948004 (127.0.0.1): No such file or directory: my_llm_env3.12/lib/python3.12/site-packages/torch/utils/data/dataloader.py
[W 2025-06-29 17:09:08.021 ServerApp] 404 GET /api/contents/my_llm_env3.12/lib/python3.12/site-packages/torch/utils/data/dataloader.py?content=0&hash=0&1751234948005 (990a2cfb35b74774811ecca73ce2f2a0@127.0.0.1) 0.48ms referer=http://localhost:8888/lab?
[W 2025-06-29 17:09:08.021 ServerApp] 404 GET /api/contents/my_llm_env3.12/lib/python3.12/site-packages/torch/utils/data/dataloader.py?content=0&hash=0&1751234948005 (127.0.0.1): No such file or directory: my_llm_env3.12/lib/python3.12/site-packages/torch/utils/data/dataloader.py
[W 2025-06-29 17:09:08.022 ServerApp] 404 GET /api/contents/my_llm_env3.12/lib/python3.12/site-packages/torch/utils/data/dataloader.py?content=0&hash=0&1751234948005 (990a2cfb35b74774811ecca73ce2f2a0@127.0.0.1) 0.75ms referer=http://localhost:8888/lab?
[W 2025-06-29 17:09:08.022 ServerApp] 404 GET /api/contents/my_llm_env3.12/lib/python3.12/site-packages/torch/utils/data/dataloader.py?content=0&hash=0&1751234948005 (127.0.0.1): No such file or directory: my_llm_env3.12/lib/python3.12/site-packages/torch/utils/data/dataloader.py
[W 2025-06-29 17:09:08.026 ServerApp] SSL Error on 15 ('2a04:4e42::223', 443, 0, 0): [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1028)
[W 2025-06-29 17:09:08.033 ServerApp] 404 GET /api/contents/projects/roof_shingle_inspection/pipeline/labels/train..?content=0&hash=0&1751234948031 (990a2cfb35b74774811ecca73ce2f2a0@127.0.0.1) 0.50ms referer=http://localhost:8888/lab?
[W 2025-06-29 17:09:08.033 ServerApp] 404 GET /api/contents/projects/roof_shingle_inspection/pipeline/labels/train..?content=0&hash=0&1751234948031 (127.0.0.1): No such file or directory: projects/roof_shingle_inspection/pipeline/labels/train..
[W 2025-06-29 17:09:08.035 ServerApp] 404 GET /api/contents/projects/roof_shingle_inspection/pipeline/labels/val...?content=0&hash=0&1751234948031 (990a2cfb35b74774811ecca73ce2f2a0@127.0.0.1) 2.24ms referer=http://localhost:8888/lab?
[W 2025-06-29 17:09:08.035 ServerApp] 404 GET /api/contents/projects/roof_shingle_inspection/pipeline/labels/val...?content=0&hash=0&1751234948031 (127.0.0.1): No such file or directory: projects/roof_shingle_inspection/pipeline/labels/val...
[I 2025-06-29 17:09:08.127 ServerApp] Kernel started: 5369a1ba-f892-406d-837f-dee2292e8c03
[I 2025-06-29 17:09:08.127 ServerApp] Kernel started: 789f7bad-fb6f-47aa-8f8c-e9dd9da936e7
[I 2025-06-29 17:09:08.131 ServerApp] Kernel started: 3dcc9341-27a5-4ce6-9b4e-3168a8749ca1
[I 2025-06-29 17:09:08.136 ServerApp] Kernel started: ebbdf0a8-011c-4f3a-b37b-73ad32dde657
[I 2025-06-29 17:09:08.370 ServerApp] Connecting to kernel 3dcc9341-27a5-4ce6-9b4e-3168a8749ca1.
[W 2025-06-29 17:09:08.370 ServerApp] The websocket_ping_timeout (90000) cannot be longer than the websocket_ping_interval (30000).
    Setting websocket_ping_timeout=30000
[I 2025-06-29 17:09:08.378 ServerApp] Starting buffering for 3dcc9341-27a5-4ce6-9b4e-3168a8749ca1:57600a7d-7415-494c-a4cd-dbbf6fe13468
[I 2025-06-29 17:09:08.384 ServerApp] Connecting to kernel 5369a1ba-f892-406d-837f-dee2292e8c03.
[I 2025-06-29 17:09:08.398 ServerApp] Connecting to kernel 789f7bad-fb6f-47aa-8f8c-e9dd9da936e7.
[I 2025-06-29 17:09:08.410 ServerApp] Connecting to kernel 3dcc9341-27a5-4ce6-9b4e-3168a8749ca1.
[I 2025-06-29 17:09:08.424 ServerApp] Connecting to kernel ebbdf0a8-011c-4f3a-b37b-73ad32dde657.
[I 2025-06-29 17:09:08.430 ServerApp] Starting buffering for ebbdf0a8-011c-4f3a-b37b-73ad32dde657:dcff7c79-e603-42e0-81a8-83dd452cd851
[I 2025-06-29 17:09:08.436 ServerApp] Connecting to kernel ebbdf0a8-011c-4f3a-b37b-73ad32dde657.
[I 2025-06-29 17:09:08.449 ServerApp] Connecting to kernel ebbdf0a8-011c-4f3a-b37b-73ad32dde657.
[I 2025-06-29 17:11:08.012 ServerApp] Saving file at /2_label_studio_setup.ipynb
[I 2025-06-29 17:11:08.017 ServerApp] Saving file at /1_vision_model_setup.ipynb
[I 2025-06-29 17:11:08.033 ServerApp] Saving file at /3_data_pipeline.ipynb
[I 2025-06-29 17:11:08.057 ServerApp] Saving file at /4_object_detection_training.ipynb
[I 2025-06-29 17:11:44.180 ServerApp] Starting buffering for 5369a1ba-f892-406d-837f-dee2292e8c03:95c4bf62-1faf-4dbe-a560-c8aa9c69c805
[I 2025-06-29 17:11:44.180 ServerApp] Starting buffering for 789f7bad-fb6f-47aa-8f8c-e9dd9da936e7:761dba2d-0668-4d64-b4a2-f4dc3c4797bd
[I 2025-06-29 17:11:44.181 ServerApp] Starting buffering for 3dcc9341-27a5-4ce6-9b4e-3168a8749ca1:5b286099-98a3-4d59-a8d2-3f7fdf568434
[I 2025-06-29 17:11:44.181 ServerApp] Starting buffering for ebbdf0a8-011c-4f3a-b37b-73ad32dde657:d6e1e8e4-ed0f-4e28-9297-e547a32ba04d
[I 2025-06-29 17:11:44.403 ServerApp] Connecting to kernel 5369a1ba-f892-406d-837f-dee2292e8c03.
[I 2025-06-29 17:11:44.411 ServerApp] Starting buffering for 5369a1ba-f892-406d-837f-dee2292e8c03:1ae4313c-5ef4-4bc1-9e45-e0696ad60dc7
[I 2025-06-29 17:11:44.414 ServerApp] Connecting to kernel 789f7bad-fb6f-47aa-8f8c-e9dd9da936e7.
[I 2025-06-29 17:11:44.420 ServerApp] Starting buffering for 789f7bad-fb6f-47aa-8f8c-e9dd9da936e7:dfc2d25c-e07b-431c-9f84-14542015dcd0
[I 2025-06-29 17:11:44.425 ServerApp] Connecting to kernel 3dcc9341-27a5-4ce6-9b4e-3168a8749ca1.
[I 2025-06-29 17:11:44.431 ServerApp] Starting buffering for 3dcc9341-27a5-4ce6-9b4e-3168a8749ca1:8e4a0f6e-9031-4186-be4d-a1c016100ee3
[I 2025-06-29 17:11:44.436 ServerApp] Connecting to kernel ebbdf0a8-011c-4f3a-b37b-73ad32dde657.
[I 2025-06-29 17:11:44.443 ServerApp] Starting buffering for ebbdf0a8-011c-4f3a-b37b-73ad32dde657:bfa1f7d4-d647-440f-93ce-97a5e94127e9
[W 2025-06-29 17:11:44.709 ServerApp] Notebook 4_object_detection_training.ipynb is not trusted
[W 2025-06-29 17:11:44.726 ServerApp] 404 GET /api/contents/my_llm_env3.12/lib/python3.12/site-packages/torch/utils/data/dataloader.py?content=0&hash=0&1751235104724 (990a2cfb35b74774811ecca73ce2f2a0@127.0.0.1) 0.61ms referer=http://localhost:8888/lab?
[W 2025-06-29 17:11:44.726 ServerApp] 404 GET /api/contents/my_llm_env3.12/lib/python3.12/site-packages/torch/utils/data/dataloader.py?content=0&hash=0&1751235104724 (127.0.0.1): No such file or directory: my_llm_env3.12/lib/python3.12/site-packages/torch/utils/data/dataloader.py
[W 2025-06-29 17:11:44.727 ServerApp] 404 GET /api/contents/my_llm_env3.12/lib/python3.12/site-packages/torch/utils/data/dataloader.py?content=0&hash=0&1751235104725 (990a2cfb35b74774811ecca73ce2f2a0@127.0.0.1) 0.64ms referer=http://localhost:8888/lab?
[W 2025-06-29 17:11:44.727 ServerApp] 404 GET /api/contents/my_llm_env3.12/lib/python3.12/site-packages/torch/utils/data/dataloader.py?content=0&hash=0&1751235104725 (127.0.0.1): No such file or directory: my_llm_env3.12/lib/python3.12/site-packages/torch/utils/data/dataloader.py
[W 2025-06-29 17:11:44.728 ServerApp] 404 GET /api/contents/my_llm_env3.12/lib/python3.12/site-packages/torch/utils/data/dataloader.py?content=0&hash=0&1751235104725 (990a2cfb35b74774811ecca73ce2f2a0@127.0.0.1) 1.04ms referer=http://localhost:8888/lab?
[W 2025-06-29 17:11:44.728 ServerApp] 404 GET /api/contents/my_llm_env3.12/lib/python3.12/site-packages/torch/utils/data/dataloader.py?content=0&hash=0&1751235104725 (127.0.0.1): No such file or directory: my_llm_env3.12/lib/python3.12/site-packages/torch/utils/data/dataloader.py
[W 2025-06-29 17:11:44.744 ServerApp] SSL Error on 102 ('2a04:4e42:400::223', 443, 0, 0): [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1028)
[I 2025-06-29 17:11:44.798 LabApp] Build is up to date
[I 2025-06-29 17:11:44.806 ServerApp] Connecting to kernel 3dcc9341-27a5-4ce6-9b4e-3168a8749ca1.
[I 2025-06-29 17:11:44.817 ServerApp] Connecting to kernel 789f7bad-fb6f-47aa-8f8c-e9dd9da936e7.
[I 2025-06-29 17:11:44.831 ServerApp] Connecting to kernel 5369a1ba-f892-406d-837f-dee2292e8c03.
[I 2025-06-29 17:11:44.845 ServerApp] Connecting to kernel ebbdf0a8-011c-4f3a-b37b-73ad32dde657.
[I 2025-06-29 17:12:13.689 ServerApp] Saving file at /4_object_detection_training.ipynb
